test/unit/spec/LoginRouter_spec.js

```diff
@@ -476,7 +476,7 @@ function (Okta, Q, Logger, OktaAuth, Util, Expect, Router,
           expect(test.afterErrorHandler).toHaveBeenCalledTimes(1);
           expect(test.afterErrorHandler.calls.allArgs()).toEqual([
             [
-              { controller: 'recovery-question' },
+              { controller: 'primary-auth' },
```

Comment:
Now, we're going to always fire the `afterError` event from the primary-auth controller, as users are navigated back to this page when the `flashError` is rendered.